dsl/jpql/src/test/kotlin/com/linecorp/kotlinjdsl/dsl/jpql/expression/IdDslTest.kt

```diff
@@ -0,0 +1,48 @@
+package com.linecorp.kotlinjdsl.dsl.jpql.expression
+
+import com.linecorp.kotlinjdsl.dsl.jpql.entity.book.Book
+import com.linecorp.kotlinjdsl.dsl.jpql.entity.book.Isbn
+import com.linecorp.kotlinjdsl.dsl.jpql.queryPart
+import com.linecorp.kotlinjdsl.querymodel.jpql.entity.Entities
+import com.linecorp.kotlinjdsl.querymodel.jpql.expression.Expression
+import com.linecorp.kotlinjdsl.querymodel.jpql.expression.Expressions
+import org.assertj.core.api.WithAssertions
+import org.junit.jupiter.api.Test
+
+class IdDslTest : WithAssertions {
+    private val entity = Entities.entity(Book::class)
+
+    @Test
+    fun `id() with a class`() {
+        // when
+        val expression = queryPart {
+            id<Isbn>(Book::class)
+        }.toExpression()
+
+        val actual: Expression<Any> = expression // for type check
```

Comment:
Is it correct to check the type with “Any”? I think it should be checked as an "Isbn" type.

Reply:
oh thank you. applied